openmetadata-service/src/main/resources/json/data/testConnections/storage/gcs.json

```diff
@@ -11,25 +11,10 @@
       "mandatory": true
     },
     {
-      "name": "ListBlobs",
```

Comment:
hi @MChamberlin why are we removing these? In order to sample the container files it's important we can list the blobs and get them I believe

Reply:
I removed these to bring the test logic in line with the existing S3 connector. I assume they were removed there because the additional tests require that state be tracked in the test client, which complicates things a bit. I can add them back though.

Reply:
hi yes I think it makes sense to make a quick check of those variables. It would also make sense adding them to S3, but that'd be work for another PR :)

we could just call the GCS client to `list_blobs`, `get_bucket` etc. to ensure all the ingredients are working before running the connector.

Thanks!

Reply:
Fixed in cd0e4e0